### PR TITLE
Added eslint, prettier

### PR DESCRIPTION
**Pull request by Ihnatenko-A**
Added eslint, prettier with halo-lab configs, lefthook. Made changes according to new configs. Made fix on home page with blogs preview, there was possible case when gatsby image could not be formed, but image source was awailable. Fixed this.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  extends: [
```
Comment (lazio) [inline .eslintrc.js:2]: do we need to keep other extends in line 17?

File: package.json [hunk 2]
```diff
@@ -42,9 +42,12 @@
     "use-breakpoints-width": "^1.0.8"
   },
   "devDependencies": {
+    "@arkweid/lefthook": "^0.7.2",
+    "@trivago/prettier-plugin-sort-imports": "^1.4.1",
     "babel-eslint": "^10.1.0",
     "cypress": "^4.5.0",
     "eslint": "^6.8.0",
+    "eslint-config-halo-lab": "^1.0.12",
     "eslint-config-prettier": "^6.11.0",
```
Comment (lazio) [inline package.json:51]: we don't need this anymore and maybe some other eslint packages can be killed

**Comment by lazio** [comment]
please create pull request to development branch

**Review by lazio — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (Ihnatenko-A) [reply .eslintrc.js:2]: Removed obsolete extends

File: package.json (hunk 2 above)
Comment (Ihnatenko-A) [reply package.json]: fixed

**Pull request closed without merging**